easybuild/easyconfigs/b/BSMAP/BSMAP-2.74_makefile-modif.patch

```diff
@@ -0,0 +1,22 @@
+Patch to point to the correct bin directory in the EASYBUILDINSTALLDIR and to
+the correct location of the ZLIB libraries
+Thekla Loizou July 2015
+--- bsmap-2.74/makefile.orig	2012-09-20 23:01:57.000000000 +0300
++++ bsmap-2.74/makefile	2014-12-16 09:24:16.033631393 +0200
+@@ -1,6 +1,6 @@
+ CC=	g++
+ 
+-BIN = $(DESTDIR)/usr/bin
++BIN=$(INSTALL_DIR)/bin
```

Comment:
well, you still need to use `installopts = "DESTDIR=%(installdir)s"` in the easyconfig files

or are you saying that `DESTDIR` is reset somehow by the Makefile itself?

if that's the case, then it's OK as-is